extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html

```diff
@@ -28,6 +28,14 @@ <h5>REST Resources</h5>
  </div>
  {/if}
  {/for}
+ <h5>Static Resources</h5>
+ <div class="row">
+  <div class="col">
+   {#for staticResources in info:staticResourcesInfo}
```

Comment:
Ah hold on, no, the `for` should be around the `row`, like for additional endpoints, no? Otherwise they all end up on the same line, no?

Reply:
oh yes, let me do that

Reply:
I did the change but after that I am getting this page - 

![image](https://user-images.githubusercontent.com/46818757/105692816-ec215180-5f24-11eb-9ad1-ada19e0680a8.png)


Then on clicking `/` ,  I am getting `http://localhost:8080/` - 
![image](https://user-images.githubusercontent.com/46818757/105692999-2e4a9300-5f25-11eb-9277-9b98f00aa4d3.png)

Reply:
Is this fine , or do I need to inspect my code again ?